src/components/assets/modals/ModalXcmBridge.vue

```diff
@@ -98,6 +99,38 @@
           </div>
         </div>
       </div>
+      <div class="container--warning">
+        <div class="row--warning">
+          <div class="column--title">
+            <span class="text--dot">・</span>
+            <span class="text--warning">{{ $t('assets.modals.xcmWarning.avoidRisk') }}</span>
+          </div>
+          <div class="icon--help">
+            <IconHelp />
+          </div>
+          <q-tooltip class="box--tooltip-warning">
+            <div>
+              <span v-if="existentialDeposit"
+                >{{
+                  $t('assets.modals.xcmWarning.tooltip', {
+                    amount: Number(existentialDeposit.amount),
+                    symbol: existentialDeposit.symbol,
+                    network: existentialDeposit.chain,
+                  })
+                }}
+              </span>
+            </div>
+          </q-tooltip>
+        </div>
+        <div class="row--warning">
+          <div class="column--title">
+            <span class="text--dot">・</span>
+            <span class="text--warning">
+              {{ $t('assets.modals.xcmWarning.fee') }}
```

Comment:
How about adding the actual fee for the transaction instead of a warning message?

Reply:
I thought that would take some time. That's good!

Reply:
I think you can do it with

```
const data = await tx.paymentInfo(signer.pair)
data.partialFee;
```

Reply:
I think `data.partialFee` returns the fee of the sending transaction. But we might want to display the deducted amount from receiving account. 
![image](https://user-images.githubusercontent.com/92044428/168850917-8e3513f3-1f55-4f7a-9d58-14fe576e3eb4.png)

Reply:
that should be the only fee there is I think? The only other method I know is https://github.com/polkadot-js/api/blob/1404994792db92f82bdeb66df85a372fbbc4ed1d/packages/rpc-augment/src/augment/jsonrpc.ts#L393 but this does not work when I tried

Reply:
> that should be the only fee there is I think?

My EVM account received only 0.0006 DOT after sending 0.001 DOT from the native account
https://www.loom.com/share/3a5a9207396d462ab6c186d5e152f4f4